cs/__init__.py

```diff
@@ -107,7 +107,8 @@ def parse_option(x):
             sys.stderr.write("\n")
             sys.exit(1)
 
-    if 'Async' not in command and 'jobid' in response and not options.async:
+    if 'Async' not in command and 'jobid' in response and not hasattr(
```

Comment:
I guess `hasattr()` will always be true in this case (option has a default value). Maybe `getattr(options, 'async')` instead?